package.json

```diff
@@ -26,16 +26,18 @@
     "express-rate-limit": "1.0.2",
     "extract-text-webpack-plugin": "0.8.0",
     "file-loader": "0.8.4",
+    "google-spreadsheet": "^1.0.1",
     "habitat": "0.4.1",
-    "hatchet": "0.3.2",
```

Comment:
I misunderstood, I didn't know the code that used hatchet was already gone.

I thought it was code we were not using at the moment, but still existed in the repo., as in a send-email endpoint that we were just not hitting anymore. Taking a closer look, and hatchet isn't even used in the repo. Totally remove it if that's the case.